backend/validators/llm.py

```
@@ -0,0 +1,141 @@
+__all__ = (
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)